crates/stdsimd-test/src/wasm.rs

```diff
@@ -0,0 +1,98 @@
+//! Disassembly calling function for `wasm32` targets.
+use wasm_bindgen::prelude::*;
+
+use ::*;
+
+#[wasm_bindgen(module = "child_process")]
+extern "C" {
+    #[wasm_bindgen(js_name = execSync)]
+    fn exec_sync(cmd: &str) -> Buffer;
+}
+
+#[wasm_bindgen(module = "buffer")]
+extern "C" {
+    type Buffer;
+    #[wasm_bindgen(method, js_name = toString)]
+    fn to_string(this: &Buffer) -> String;
+}
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(js_namespace = require)]
+    fn resolve(module: &str) -> String;
```

Comment:
```
LinkError: WebAssembly Instantiation: Import #19 module="./wasm-bindgen-test" function="__wbg_resolve_06ea196b351af635" error: function import requires a callable
    at Object.<anonymous> (/checkout/target/wasm32-unknown-unknown/wbg-tmp/wasm-bindgen-test_bg.js:8:22)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/checkout/target/wasm32-unknown-unknown/wbg-tmp/wasm-bindgen-test.js:575:8)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
```

cc @alexcrichton Is this just because I moved them into a module?